ios/Mattermost/Info.plist

```diff
@@ -32,6 +32,7 @@
 			<key>CFBundleURLSchemes</key>
 			<array>
 				<string>mattermost</string>
+				<string>mmauthbeta</string>
```

Comment:
Please remove, this is handled by the build pipeline

Reply:
corrected.

Reply:
@enahum  It was required for the deep-linking into the app and it's also present on v1.  It is a key that is added in during development ?

Reply:
ok I missed this was on v1, so is ok to keep